src/customisations/models/IMediaEventContent.ts

```diff
@@ -47,6 +47,7 @@ export interface IMediaEventInfo {
 
 export interface IMediaEventContent {
```

Comment:
I'm a little confused why this interface is here instead of in the js-sdk, but I guess that doesn't have to be a problem for right now